src/Microsoft.DotNet.Watcher.Tools/project.json

```diff
@@ -1,5 +1,12 @@
 {
   "version": "1.0.0-*",
+  "description": "Dotnet watch CLI command.",
```

Comment:
Take some inspiration from this one: https://github.com/aspnet/Caching/blob/dev/src/Microsoft.Extensions.Caching.SqlConfig.Tools/project.json#L9